test/unit/modules/helpers/get-n-parents-spec.js

```diff
@@ -20,7 +19,7 @@ describe('RecordingLibrary.helpers.getNParents', () => {
   const p5 = document.createElement('h1');
   p5.setAttribute('title', 'h1title');
 
-  const p6 = document.createElement('body');
+  const p6 = document.body;
```

Comment:
When the Jest config uses the `jsdom` environment like ours does, `document.body` exists already, so you don’t need to recreate it. If you do recreate it, you have to take the extra step of actually attaching it to the document (which is not trivial) or it can cause obscure problems with code traversing the DOM :sweat_smile:

These specs started failing for me because the`body` element wasn’t attached to the DOM, and so its parent node was null, and this caused null pointer exceptions when attempting to traverse the parent tree to generate the unique selector. This doesn’t happen to real `body`'s (their parent node is the `html` node, whose parent is the `#document` itself)

The fix was as simple as this, but took hours to debug :persevere:

The other changes in various test files to append test nodes to the document during test setup are for similar reasons.